meetups/index.md

```diff
@@ -5,18 +5,29 @@ title: Meetups
 ---
 
 * [35th Meetup](https://www.meetup.com/PyData-London-Meetup/events/240813803/)
-  * Richard J Brooker on AI & ART - Style Transfer
-  * Saeed Amen on Using Python in financial markets
-  * Won Bae Suh on Arms for Sale: UK Bazzar in the 21st Century
+  * Richard J Brooker on [AI & ART - Style Transfer](https://www.youtube.com/watch?v=U1vb-NI9sKE)
+  * Saeed Amen on [Using Python in financial markets](https://www.meetup.com/PyData-London-Meetup/events/240813803/comments/481296769/)

```

Comment:
Sure, will change tonight when I'm out the office. As far as I can see the slides haven't been uploaded (please correct me if I am wrong), I'll link to the github content and examples directly though!